src/connectors/panoptic/panoptic.ts

```diff
@@ -318,18 +316,18 @@ export class Panoptic {
     longCallStrike: number, 
     straddleStrike: number, 
     asset: BigNumber,
-  ): Promise<BigNumber | unknown> {
+  ): Promise<any> {
```

Comment:
Is it possible that we could change all these `Promise<any>` to `Promise<ethers.Transaction>` or something like that?

Reply:
So I can use the ethers `ContractTransaction` and `ContractReceipt` interfaces for state-changing transactions. But for read-only transactions, I'll need to use the known output types expressed in the ABI. This is fine, but is it safe to assume every method on the ABI with no output specified is state-changing, and that every method with output specified is read-only?

Reply:
Done: f334a2d6e054979a6dd42eab8c4ca7a15c5aa8d4